Rakefile

```diff
@@ -8,3 +8,8 @@ task :rainy_day => :environment do
   RainyDayJob.set.perform_now(:just_set_up)
   puts "let the texting begin"
 end
+
+task :generate_waterings => :environment do
+  Scheduler.generate_waterings_for_a_week_from_today
```

Comment:
Super cool that Heroku can call our rake tasks

Reply:
🌟